vault/plan.sh

```diff
@@ -1,4 +1,4 @@
-pkg_origin=core
+pkg_origin=dbhagen
```

Comment:
Quick note - I haven't done a full review of this plan yet but easy fix, you'll want to switch this back to core